src/Data/Int.purs

```diff
@@ -77,6 +79,21 @@ foreign import toNumber :: Int -> Number
 fromString :: String -> Maybe Int
 fromString = fromStringAs (Radix 10)
 
+-- | A type for describing whether an integer is even or odd.
+data Parity = Even | Odd
+
+derive instance eqParity :: Eq Parity
+derive instance ordParity :: Ord Parity
```

Comment:
I think the other one just flips 0 and 1 then? In which case I guess they're isomorphic as sets but not as rings because the mapping doesn't take 0 to 0 and 1 to 1.